go/porcelain/functions_manifest.go

```diff
@@ -7,16 +7,17 @@ type functionsManifest struct {
 }
 
 type functionsManifestEntry struct {
-	MainFile       string          `json:"mainFile"`
-	Name           string          `json:"name"`
-	Path           string          `json:"path"`
-	Runtime        string          `json:"runtime"`
-	RuntimeVersion string          `json:"runtimeVersion"`
-	Schedule       string          `json:"schedule"`
-	DisplayName    string          `json:"displayName"`
-	Generator      string          `json:"generator"`
-	InvocationMode string          `json:"invocationMode"`
-	Routes         []functionRoute `json:"routes"`
+	MainFile       string                 `json:"mainFile"`
+	Name           string                 `json:"name"`
+	Path           string                 `json:"path"`
+	Runtime        string                 `json:"runtime"`
+	RuntimeVersion string                 `json:"runtimeVersion"`
+	Schedule       string                 `json:"schedule"`
+	DisplayName    string                 `json:"displayName"`
+	Generator      string                 `json:"generator"`
+	BuildData      map[string]interface{} `json:"buildData"`
```

Comment:
It's subject to change -- we wanted to make it extensible so that when we add more properties in the future we don't have to add them to every repo in the chain